src/woo_publications/publications/api/serializers.py

```diff
@@ -171,6 +180,39 @@ def validate_publicatiestatus(
 
         return value
 
+    def get_or_create_organisation_member(self, owner: EigenaarSerializer | None):
```

Comment:
`owner` will not be an instance of `EigenaarSerializer`, it will be be a dictionary with the relevant keys + values.